packages/next/src/seo/utils/get-clean-excerpt.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Removes all HTML tags from a given string.
+ *
+ * @param html - The HTML string to be stripped.
+ * @return The cleaned text without HTML tags.
+ */
+export default function getCleanExcerpt( html: string ): string {
```

Comment:
If we really need this function, probably better to call it `sanitizeHtml()` and to export it from `@/core`